make/crosscompile.jetson.mk

```diff
@@ -71,15 +71,20 @@ USE_CUDA = 1
 
 # add the path to CUDA library to link and compile flag
 # if you have already add them to environment variable, leave it as NONE
-# USE_CUDA_PATH = /usr/local/cuda
-USE_CUDA_PATH = /usr/local/cuda-9.0/targets/aarch64-linux
+USE_CUDA_PATH = /usr/local/cuda
 
 # whether to enable CUDA runtime compilation
 ENABLE_CUDA_RTC = 0
 
 # whether use CuDNN R3 library
 USE_CUDNN = 1
 
+# set flags for jetson boards
+CUDA_ARCH := -gencode arch=compute_53,code=sm_53 \
+             -gencode arch=compute_62,code=sm_62
```

Comment:
do we have 6.2 on Jetsons?

Reply:
Yeah, this is what the Jetson Nano docs specify.